admin/data/sql/migrations/20190528131913.do.remove-checkdata-from-vewcompletedcheckcount.sql

```diff
@@ -0,0 +1,6 @@
+ALTER VIEW [mtc_admin].[vewCompletedCheckCount] AS
+  SELECT c.pupil_id, COUNT(c.id) AS [CompletedCheckCount]
+  FROM [mtc_admin].[check] c
+    LEFT JOIN [mtc_admin].checkResult ck ON (c.id = ck.check_id)
```

Comment:
NB: I am surprised that `ck` is short for the `c`heck `r`esult table.  Why not use `cr`?